src/nodes.coffee

```diff
@@ -1728,6 +1771,21 @@ exports.Assign = class Assign extends Base
         # destructured variables.
         @variable.base.lhs = yes
         return @compileDestructuring o unless @variable.isAssignable()
+        # Find rest elements in object destructuring. Can be removed once ES proposal hits Stage 4.
```

Comment:
Actually, I started to work on a version before, where I tried to achieve something like that, but then I hit some walls and ended with this. 
I'll try to refactor this.